src/resources/parser/glb-parser.js

```diff
@@ -816,7 +871,7 @@ const createMesh = function (device, gltfMesh, accessors, bufferViews, callback,
 
                     if (target.hasOwnProperty('POSITION')) {
                         accessor = accessors[target.POSITION];
-                        options.deltaPositions = getAccessorData(accessor, bufferViews);
+                        options.deltaPositions = getAccessorData(accessor, bufferViews, true);
                         options.deltaPositionsType = getComponentType(accessor.componentType);
                         if (accessor.hasOwnProperty('min') && accessor.hasOwnProperty('max')) {
                             options.aabb = new BoundingBox();
```

Comment:
when looking at other engines, I've noticed they dequantize min and max bounds of both mesh and morph targets - perhaps that should be added as well.

Reply:
thanks good point. done.